### ISSUE TITLE
GPX export is not working in the app

### ISSUE THREAD

**Issue by 0av** [boldtrn/graphhopper-maps-capacitor#28]
GPX export is not working in the app.

Version: 1.0.5
Repo.: F-Droid

Android Version: 13
Lineage OS Version: 20-20230709-NIGHTLY-hotdogb

**Comment by boldtrn** [comment]
We would need to use some Capacitor Plugin to save files. One possible approach would be: https://capacitorjs.com/docs/apis/filesystem

Or we could open the GPX download in the browser?

**Comment by boldtrn** [comment]
Yes, but this is a JS "download". I mean like it worked in the old gh maps, where a new tab was opened. If we would open this tab in the browser and not in the app, we can use the file access of the browser to store the file, otherwise we have to implement this storing logic ourselves.

**Comment by boldtrn** [comment]
I guess all options would require changes in the JS code. 

> The old GH maps forwarded the request to the server which triggered the download. Is this what would be required?

This would be the easiest option IMHO.

**Comment by boldtrn** [comment]
Just to get a better understanding, why do we need the experimental chrome API? If we need to overwrite something in app.js anyway, why not check for some global function that is registered in the app.js and call that if it is available?

**Comment by JonnyTech** [comment]
Unable to get this working on version 2.6.0 from f-droid. Is there any progress on GPX file downloads from the app?
